wiki/osu!tourney/en.md

```diff
@@ -7,285 +7,22 @@ Users encountering **problems** with the client may read the Troubleshooting sec
 ## What is osu!tourney?
 
 The osu!tourney client is the official streaming client used to spectate all players in a multiplayer room simultaneously.
-It is, currently, only meant to be used for world cups and might therefore be limited in its behaviour.
+It is, currently, only meant to be used for world cups and might therefore be limited in its behaviour. (You can still use it personally though, read multiplayer usage for more info.)
 
 The client, henceforth referred to as _osu!tourney_ supports matches with 2 teams of up to 8 players each.
 
-## Setup
+### Index
```

Comment:
remove this section and its contents, replace it with:
```
## Related articles

- [Setup](/wiki/osu!tourney/Setup/ "Setup")
- [Multiplayer Usage](/wiki/osu!tourney/Multiplayer_Usage/ "Multiplayer session creation and handling")
- [Command List](/wiki/osu!tourney/Tournament_Management_Commands/ "Tournament Management Commands/")
- [Skinning](/wiki/osu!tourney/Skinning/ "Skinning")
- [Troubleshooting](/wiki/osu!tourney/Troubleshooting/ "Troubleshooting")
- [Prizes](/wiki/osu!tourney/Prizes/ "Prizes")
```

Reply:
if https://github.com/ppy/osu-wiki/pull/633#issuecomment-325170562 is the case, then:
```
## Related articles

- [Setup](/wiki/osu!tourney/Setup/ "Setup")
- [Multiplayer Usage](/wiki/osu!tourney/Multiplayer_Usage/ "Multiplayer session creation and handling")
- [Command List](/wiki/osu!tourney/Tournament_Management_Commands/ "Tournament Management Commands/")
- [Skinning](/wiki/osu!tourney/Skinning/ "Skinning")
- [Tournament Management Commands](/wiki/osu!tourney/Tournament_Management_Commands "Tournament Management Commands")
- [Troubleshooting](/wiki/osu!tourney/Troubleshooting/ "Troubleshooting")
- [Prizes](/wiki/osu!tourney/Prizes/ "Prizes")
```